packages/config-service/src/services/loggerService.ts

```diff
@@ -28,14 +28,28 @@ export class LoggerService {
     GlobalConfig.ENTRIES.GH_ACCESS_TOKEN.envName,
   ];
 
+  public static readonly KNOWN_SECRET_PREFIXES = [
+    'ghp', // GitHub personal access tokens
+    'gho', // OAuth access tokens
+    'ghu', // GitHub user-to-server tokens
+    'ghs', // GitHub server-to-server tokens
+    'ghr', // refresh tokens
+  ];
+
   /**
    * Hide sensitive information
    *
    * @param envName
    * @param envValue
    */
   static maskUpEnv(envName: string, envValue: string | undefined): string {
-    if (this.SENSITIVE_FIELDS.indexOf(envName) > -1) {
+    const isSensitiveField: boolean = this.SENSITIVE_FIELDS.indexOf(envName) > -1;
+    const isKnownSecret: boolean =
+      GlobalConfig.ENTRIES[envName].type === 'string' && envValue
+        ? !!envValue.match(/^(gh[pousr]_[a-zA-Z0-9]{36,251}|github_pat_[a-zA-Z0-9]{22}_[a-zA-Z0-9]{59})$/)
```

Comment:
For readability it would make sense to break this regex out into a constant that can be easily read and updated in the future if need be:
` GITHUB_SECRET_PATTERN: RegExp = /^(gh[pousr]_[a-zA-Z0-9]{36,251}|github_pat_[a-zA-Z0-9]{22}_[a-zA-Z0-9]{59})$/; `

then:
` ... match(this.GITHUB_SECRET_PATTERN)`

Reply:
Fixed.